app/controllers/auth0_sign_in_callback.js

```diff
@@ -16,23 +16,28 @@ const AccessTokenHandler = function (req, res) {
     try {
       const user = await auth0.getProfile(body.access_token)
       const apiRequestBody = getUserInfo(user)
-
       //  Must be an absolute URI
       const endpoint =
         config.restapi.protocol +
         config.app_host_port +
         config.restapi.baseuri +
         '/v1/users'
+
       axios
         .post(endpoint, apiRequestBody)
         .then((response) => {
           const body = response.data
-          res.cookie('user_id', body.id)
+
+          // TODO add code to properly resolve user_id
```

Comment:
Can you describe the context for the TODO in fuller detail in the comment? What's happening with `user_id` that isn't resolved properly?

Reply:
I've updated this, it now says `// TODO resolve user via auth0`, let me know if you'd like me to remove it. We can set a cookie here for the auth0 library to pick up, that re-write is coming soon 👀